pom.xml

```diff
@@ -10,7 +10,7 @@
   <artifactId>plugin</artifactId>
   <!-- Baseline Jenkins version you use to build and test the plugin. Users 
    must have this version or newer to run. -->
-  <version>2.8</version>
+  <version>1.609.3</version>
```

Comment:
I tested this by looking at the debug log messages during a compile:

```
[shebert@localhost foreman-node-sharing]$ mvn -X clean compile -DskipTests  | grep jenkins | grep war
```

`
[DEBUG]    org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]    org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]   testArtifact: artifact=org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]   includeArtifact: artifact=org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]   startProcessChildren: artifact=org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]   endProcessChildren: artifact=org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]   testArtifact: artifact=org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]   includeArtifact: artifact=org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]   startProcessChildren: artifact=org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]   endProcessChildren: artifact=org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]    org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]    org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]   testArtifact: artifact=org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]   includeArtifact: artifact=org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]   startProcessChildren: artifact=org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]   endProcessChildren: artifact=org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]   testArtifact: artifact=org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]   includeArtifact: artifact=org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]   startProcessChildren: artifact=org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]   endProcessChildren: artifact=org.jenkins-ci.main:jenkins-war:war:1.651.3:test
[DEBUG]    org.jenkins-ci.main:jenkins-war:jar:war-for-test:1.651.3:test
[DEBUG]    org.jenkins-ci.main:jenkins-war:war:1.651.3:test
`

AND

```
shebert@localhost foreman-node-sharing]$ mvn -X clean compile -DskipTests  | grep jenkins | grep core
        <exclude>org.jenkins-ci.main:jenkins-core</exclude>
        <exclude>org.jenkins-ci.main:jenkins-core</exclude>
[DEBUG]    org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]   (f) excludes = [org.jenkins-ci.main:jenkins-core, org.jenkins-ci.main:cli, org.jenkins-ci.main:jenkins-test-harness, com.google.code.findbugs:annotations]
[DEBUG]   testArtifact: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]   includeArtifact: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]   startProcessChildren: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]   endProcessChildren: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]     manageArtifactVersion: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:test, replacement=org.jenkins-ci.main:jenkins-core:jar:1.651.3
[DEBUG]     testArtifact: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:test
[DEBUG]     manageArtifactVersion: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:test, replacement=org.jenkins-ci.main:jenkins-core:jar:1.651.3
[DEBUG]     omitForNearer: omitted=org.jenkins-ci.main:jenkins-core:jar:1.651.3:test kept=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]    org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]   (f) excludes = [org.jenkins-ci.main:jenkins-core, org.jenkins-ci.main:cli, org.jenkins-ci.main:jenkins-test-harness, com.google.code.findbugs:annotations]
[DEBUG]   testArtifact: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]   includeArtifact: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]   startProcessChildren: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]   endProcessChildren: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]     manageArtifactVersion: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:test, replacement=org.jenkins-ci.main:jenkins-core:jar:1.651.3
[DEBUG]     testArtifact: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:test
[DEBUG]     manageArtifactVersion: artifact=org.jenkins-ci.main:jenkins-core:jar:1.651.3:test, replacement=org.jenkins-ci.main:jenkins-core:jar:1.651.3
[DEBUG]     omitForNearer: omitted=org.jenkins-ci.main:jenkins-core:jar:1.651.3:test kept=org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
[DEBUG]    org.jenkins-ci.main:jenkins-core:jar:1.651.3:provided
```

There is no mention of 2.X for either.

Therefore, we can change the jenkins.version to be 1.609.3 for now but I would prefer to keep the parent pom at 2.12 (latest)